cinch/roles/jenkins_master/tasks/https.yml

```diff
@@ -15,34 +15,20 @@
     mode: 0755
   with_items:
     - in: "{{ jenkins_ssl_cert }}"
-      out: jenkins.crt
+      out: "jenkins.crt"
     - in: "{{ jenkins_ssl_key }}"
-      out: jenkins.key
+      out: "jenkins.key"
```

Comment:
In YAML strings only need to be quoted if they are ambiguous - e.g. if they start with a quotation mark or single tick that needs to be part of the string, if it starts with a YAML control character such as '{' or '[', etc. When a string literal does not begin with any of those, then the quotation marks can be left out like we do in all the other locations.